client/src/pages/AddFood.js

```diff
@@ -35,46 +38,45 @@ function AddFood() {
   });
 
   function handleForm(e) {
-    e.preventDefault();
     mutation.mutate(url, { title: "title", img: "img", author: "author" });
   }
 
-  function handleChange(e) {
-    setFood(e.target.value);
-    console.log(e);
-  }
-
   return (
     <div className="form">
       <Layout />
       <Form onSubmit={handleSubmit((data) => setData(JSON.stringify(data)))}>
-        <label>Food Name</label>
+        <label>Food Title</label>
+        <input
+          type="text"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+        <label>Food Author</label>
+
         <input
           type="text"
-          {...register("foodName")}
-          placeholder="Food name"
-          onChange={handleChange}
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
         />
+
         <label>Food Type</label>
         <select {...register("author", { required: true })}>
-          <option value="Breakfast">Breakfast.</option>
+          <option value="Breakfast">Breakfast</option>
           <option value="Lunch">Lunch</option>
           <option value="Dinner">Dinner</option>
         </select>
         <label>Food Description</label>
-        <textarea {...register("title")} placeholder="About food" />
-        <p>{data}</p>
-        <Button variant="contained" onClick={handleForm}>
-          Submit
-        </Button>
+        <textarea value={img} onChange={(e) => setImage(e.target.value)} />
```

Comment:
Did you mean to have image path management here in this textarea?  Or was this meant to be the Food item's description?